src/components/SingleSelect/SingleSelect.tsx

```diff
@@ -254,7 +360,7 @@ const SingleSelect = createClass({
 			<DropMenu
 				{...dropMenuProps}
 				isDisabled={isDisabled}
-				selectedIndices={isItemSelected ? [selectedIndex] : []}
+				selectedIndices={isItemSelected ? [selectedIndex as number] : []}
```

Comment:
I double checked and this seems fine since it's already been run through `isNumber`

Reply:
I get an error if I remove the `as number`, typescript doesn't seem to pick up the `isNumber` check through `isItemSelected`.